lib/loaders/per_type.js

```diff
@@ -14,12 +15,15 @@ const apiLoader = (api, path) => {
 }
 const gravityLoader = apiLoader.bind(null, gravity)
 const positronLoader = apiLoader.bind(null, positron)
-
 export default () => {
   return {
     articlesLoader: positronLoader("articles"),
     artistLoader: gravityLoader(id => `artist/${id}`),
     artworkLoader: gravityLoader(id => `artwork/${id}`),
+    artworksLoader: gravityLoader(ids => {
```

Comment:
Ah, hmm, might be because the way we currently wrap DataLoader is in a sort of transitional phase. Ok, in this case maybe let’s just revert to how you were doing it.